File: app/src/main/java/com/karthik/splash/Models/PhotoDetail/Urls.java
package com.karthik.splash.Models.PhotoDetail;


import com.google.gson.annotations.SerializedName;


public class Urls{

	@SerializedName("small")
	private String small;

	@SerializedName("thumb")
	private String thumb;

	@SerializedName("raw")
	private String raw;

	@SerializedName("regular")
	private String regular;

	@SerializedName("full")
	private String full;
}

File: app/src/main/java/com/karthik/splash/models/likephoto/Urls.kt
package com.karthik.splash.models.likephoto


import com.google.gson.annotations.SerializedName

data class Urls(
        @SerializedName("small") private val small: String? = null,
        @SerializedName("thumb") private val thumb: String? = null,
        @SerializedName("raw") private val raw: String? = null,
        @SerializedName("regular") private val regular: String? = null,
        @SerializedName("full") private val full: String? = null
)
